src/components/v5/shared/MembersSelect/types.ts

```diff
@@ -1,5 +1,5 @@
 import { User } from '~types';
 
 export interface MemberSelectProps {
-  user?: User;
+  user?: User | null;
```

Comment:
why null?

Reply:
It's null because this type comes from graphql and it's adding null by default